integration_tests/README.md

```diff
@@ -37,20 +37,20 @@ It is recommended that you use `pyenv` to manage Python installations.
 - Follow instructions to use the right method of installation described
   [here](https://github.com/pyenv/pyenv#installation)
 - Verify that `pyenv` is set correctly
-  
```

Comment:
Why is this file being changed when it does not appear to be related to anything else in the PR?

Reply:
To explain how the user should translate a Spark config key with an underscore to the environment variable for integration tests

https://github.com/NVIDIA/spark-rapids/pull/6358/files#diff-2fd87c8f6ba9eceb00af05de7edefb7b5e70706cd27bebf9d923da293af4476bR232-R235

The rest is my editor trimming trailing white spaces . You can hit "Hide whitespace"